optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java

```diff
@@ -51,7 +51,7 @@
     }
 
     /**
-     * The source variables (masters) that trigger a change to this shadow variable (slave).
+     * The source variables (leaders) that trigger a change to this shadow variable (follower).
```

Comment:
Please do not make changes to terminology without discussing with the developer team. 
Please remove this change from this PR, and let's dicsuss properly how to change this wording.

Reply:
I checked this is the terminology on the `main` branch, assuming it has been properly reviewed and accepted there.

Reply:
so this change to keep is ok?